### PR TITLE
q1

### PR DESCRIPTION
**Pull request by victorFFFF**

### CODE REVIEW

File: victorLi/quiz/q1.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
```
Comment (knrodriguez) [inline victorLi/quiz/q1.js:2]: Missing approach and edge cases.

File: victorLi/quiz/q1.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    if( tempSum > maxSum)
+    {
+      maxSum = tempSum
+      result = [ array[begin] , array[begin+1] , array[end]]
+    }
+     
+    end++
+    begin++
+  }
+  return result
```
Comment (knrodriguez) [inline victorLi/quiz/q1.js:28]: This function should return the maximum sum, not the subarray that gives you the maximum sum.

File: victorLi/quiz/q1.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+
+let k = 3
+console.log(maxSum(x,k))
+console.log(maxSum(y,k))
+console.log(maxSum(z,k))
```
Comment (knrodriguez) [inline victorLi/quiz/q1.js:35]: Missing test cases that assert edge cases.

**Review by knrodriguez — APPROVED** [review]

**Comment by knrodriguez** [comment]
A high level approach will help interviewers understand how your code solves the prompt, as well as helps you understand what is being asked of you. Test cases and edge cases show the interviewer that you are thinking critically of the problem.
